### ISSUE TITLE
Displaying dicts in the admin console does not render nicely

### ISSUE THREAD

**Issue by alexk307** [tomchristie/django-rest-framework#4532]
## Checklist
- [x] I have verified that that issue exists against the `master` branch of Django REST framework.
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] This is not a usage question. (Those should be directed to the [discussion group](https://groups.google.com/forum/#!forum/django-rest-framework) instead.)
- [x] This cannot be dealt with as a third party library. (We prefer new functionality to be [in the form of third party libraries](http://www.django-rest-framework.org/topics/third-party-resources/#about-third-party-packages) where possible.)
- [x] I have reduced the issue to the simplest possible case.
- [] I have included a failing test as a pull request. (If you are unable to do so we can still accept the issue.)

There is no [template](https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/templates/rest_framework/admin/dict_value.html) code for displaying dicts in the admin console. Ideally this should mimic some of other templates for the admin console as to display the dict in a nice way.

**Comment by rocktavious** [comment]
FWIW - I use this template override for  `templates/rest_framework/admin/dict_value.html`

```
{% load rest_framework %}
<table class="table table-striped">
  <tbody>
    {% for key, value in value.items %}
      <tr>
        <th>{{ key|format_value }}</th>
        <td>{{ value|format_value }}</td>
      </tr>
    {% endfor %}
  </tbody>
</table>
```

**Comment by alexk307** [comment]
That should work fix the issue. I think all that's needed is a template, and this looks like it'll do the job.

**Comment by rocktavious** [comment]
I also changed the `templates/rest_framework/admin/list_value.html`
to give it a little bit better view when working with nested dictionaries
```
{% load rest_framework %}
  {% for item in value %}
  <div class="well">
      {{ item|format_value }}
  </div>
  {% endfor %}
```

i found the `well` class from bootstrap did a better job then using another table with an auto incremented index value

example:
![screen shot 2016-10-27 at 12 26 44 pm](https://cloud.githubusercontent.com/assets/735015/19778082/c1f999e0-9c40-11e6-8470-9a65a04ace2c.png)

**Comment by tomchristie** [comment]
Looks promising. Pull requests welcome. Otherwise will review this in due course.

**Comment by alexk307** [comment]
@rocktavious happy to do it if you don't want to!

**Comment by rocktavious** [comment]
Currently don't have the bandwidth - still trying to upgrade all our projects at work to DRF 3.5 and have alot of things to fixup.  So if you got the time and drive feel free to take it over @alexk307 !

**Comment by alexk307** [comment]
@rocktavious I'll take care of it. Cheers!

**Comment by alexk307** [comment]
took a stab here: https://github.com/tomchristie/django-rest-framework/pull/4636

**Comment by tomchristie** [comment]
Closed via #4636 - great stuff!
